build.user.properties.template

```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
```

Comment:
Ugh yeah, we need to have a generator script that creates the appropriate properties file then. I didn't think about the jenkins problem.